notebooks/Model_Training_with_BERT.ipynb

```diff
@@ -0,0 +1,5371 @@
+{
```

Comment:
<p>This cell can be merged with the next cell. Also, you can move the initialization of <code>bert</code>  down to the later cell where that object is first used.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CODAIT/text-extensions-for-pandas/pull/116/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>